src/ohri-form.component.tsx

```diff
@@ -210,14 +211,17 @@ const OHRIForm: React.FC<OHRIFormProps> = ({
                       }
                     });
                     if (encounterData.length) {
-                      await postAction.applyAction(
-                        {
-                          patient,
-                          sessionMode,
-                          encounters: encounterData,
-                        },
-                        config,
-                      );
+                      const isActionEnabled = enabled ? isPostSubmissionEnabled(enabled, encounterData) : true;
```

Comment:
Was logic meant to take care of backwards compatibility? if am reading this correctly if translates to if `enabled (true)` then do an action otherwise return `true` . So in both cases it resolves to `true`. If the condition `enabled` only applies to programs, shouldn't we consider adding it within the that handler to skip this step?

The main approach here assuming it's to either enroll or dis-enroll

Reply:
Hi @pirupius, if you remember that call we had with @ebambo, the `post submission action` is by default enabled, however it may be disabled given some conditions. See example below:
 ```
 "postSubmissionActions": [
    {
      "actionId": "ProgramEnrollmentSubmissionAction",
      "enabled":"tbProgramType === '160541AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA'",
      "config": {
      "enrollmentDate": "tbRegDate",
      "programUuid": "58005eb2-4560-4ada-b7bb-67a5cffa0a27",
      "completionDate": "outcomeTBRx"
    }
  },
  {
    "actionId": "ProgramEnrollmentSubmissionAction",
    "enabled":"tbProgramType === '160052AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA'",
    "config": {
    "enrollmentDate": "tbRegDate",
    "programUuid": "00f37871-0578-4ebc-af1d-e4b3ce75310d",
    "completionDate": "outcomeTBRx"
  }
}
  ],
```

We have 2 `post submission actions` but only one can run depending on the program type selected. The other will return `false` for the `enabled flag`